benchmarks/experiment_runner.py

```diff
@@ -285,6 +285,7 @@ def run_once_and_gather_metrics(self, benchmark_experiment, benchmark_model,
     # Reset state and sync.
     reset_rng_state(benchmark_experiment)
     if benchmark_experiment.torch_xla2:
+      import jax
```

Comment:
Why do we need to import `jax`, here?

Reply:
Thanks for catching that, relocated to the right place.